python-lib/gluonts_forecasts/custom_models/autoarima.py

```diff
@@ -102,7 +103,10 @@ def train(self, training_data, validation_data=None):
             kwargs = self._set_seasonality(self.kwargs, item[TIMESERIES_KEYS.TARGET])
             external_features = self._set_external_features(kwargs, item)
 
-            model = pm.auto_arima(item[TIMESERIES_KEYS.TARGET], X=external_features, **kwargs)
+            with threadpool_limits(limits=1, user_api='blas'):
```

Comment:
Are we certain the user_api is blas only?
Can we make limits a parameter of the model, with a default at 1?

Reply:
964894d